iothub/device/src/Authentication/IotHubConnectionCredentials.cs

```diff
@@ -262,28 +262,28 @@ private void SetTokenRefresherIfApplicable()
             {
                 if (ModuleId.IsNullOrWhiteSpace())
                 {
-                    SasTokenRefresher = new DeviceAuthenticationWithSakRefresh(
-                        DeviceId,
-                        SharedAccessKey,
-                        SharedAccessKeyName,
-                        SasTokenTimeToLive,
-                        SasTokenRenewalBuffer);
+                    SasTokenRefresher = new ClientAuthenticationWithSakRefresh(
```

Comment:
Why use parameter names?

Reply:
so we don't have to pass null as `moduleId` in here. Or maybe we like to pass null?